src/main/java/com/the/hugging/team/utils/Dialogs.java

```diff
@@ -265,16 +269,16 @@ public static void cashRegisterHistoryDialog(CashRegister cr) {
             dialog.setHeaderText(null);
             dialog.getDialogPane().setPrefSize(500, 600);
 
-            FilteredList<Transaction> filteredList = new FilteredList<>(data, p -> true);
+            FilteredList<Sale> filteredList = new FilteredList<>(data, p -> true);
```

Comment:
If there is no search and need for the data to be filtered, I don't think there is need for FilteredList. You can just use the ObservableList.